thanos/charts/alerts/store.yaml

```diff
@@ -0,0 +1,81 @@
+groups:
+- name: thanos-store
```

Comment:
please same severity as in

https://github.com/sapcc/helm-charts/blob/f0fc39993f636598cd2b064408dc515e73f2dcab/common/thanos/templates/alerts/_thanos-store.alerts.tpl